modules/exploits/multi/http/glassfish_deployer.rb

```diff
@@ -76,27 +55,31 @@ def initialize(info={})
           OptString.new('APP_RPORT',[ true,  'The Application interface port', '8080']),
           OptString.new('USERNAME', [ false, 'The username to authenticate as','admin' ]),
           OptString.new('PASSWORD', [ false, 'The password for the specified username','' ]),
-          OptString.new('PATH', [ true,  "The URI path of the GlassFish Server", '/'])
+          OptString.new('PATH', [ true,  "The URI path of the GlassFish Server", '/']),
```

Comment:
Maybe we can use the opportunity to switch to TARGET_URI? I undrestand maybe out of scope, the PR already introducing a bunch of changes :)